helix-lsp/src/snippet.rs

```diff
@@ -395,22 +420,27 @@ mod parser {
         )
     }
 
-    fn anything<'a>(escape_chars: &'static [char]) -> impl Parser<'a, Output = SnippetElement<'a>> {
+    fn anything<'a>(
+        escape_chars: &'static [char],
+        end_at_brace: bool,
```

Comment:
Ah sorry, I meant to answer this but somehow my comment didn't get posted. Even with const generics this is not that easy to do since we need to add `$` to the slice (which we need to do). It's possible to do this with a bunch of fancy array tricks + const generic magic  (or by allocating a vec) but I think a simple  `bool` parameter is preferable to that. If we actually need something else than a bracket here we can revisit